spec/index.bs

```diff
@@ -1509,9 +1507,18 @@ The Trusted Types portion of this algorithm uses |calleeRealm| and its CSP setti
 </pre>
 </div>
 
-Given a [[ECMA-262#realm|realm]] (|calleeRealm|), a list of strings (|parameterStrings|), a string (|bodyString|), <ins> a string (|source|), an enum (|compilationType|), a list of ECMAScript language values (|parameterArgs|), and an ECMAScript language value (|bodyArg|), this algorithm returns normally if compilation is allowed, and
+Given a [[ECMA-262#realm|realm]] (|calleeRealm|), a list of strings (|parameterStrings|), a string (|bodyString|), <ins> an enum (|compilationType|), a list of ECMAScript language values (|parameterArgs|), and an ECMAScript language value (|bodyArg|)</ins>, this algorithm returns normally if compilation is allowed, and
 throws an "`EvalError`" if not:
 
+1.  <ins>Let |source| be |bodyString|.</ins>
+1.  <ins>If |compilationType| is `*FUNCTION*`:</ins>
+    1.  <ins>Set |source| to `"function anonymous("`</ins>
```

Comment:
This matches WebKit and Firefox but not Chrome who wrap these in brackets for some reason.

These also lose aync, and generator context but I imagine their constructors are practically never actually invoked.